src/shared.ts

```diff
@@ -147,14 +147,17 @@ export const initStripeConnect = (
         for (const method in methods) {
           (element as any)[method] = function(value: any) {
             stripeConnectInstance.then(() => {
-              this[`${method}InternalOnly`](value);
+              if (this[`${method}InternalOnly`]) {
```

Comment:
Would't this approach potentially cause an issue where the method does not get called at all?

Reply:
(same for setConnector)